dev-lang/julia/julia-1.4.0.ebuild

```diff
@@ -0,0 +1,175 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+RESTRICT="splitdebug"
+
+inherit llvm pax-utils toolchain-funcs
+
+MY_PV="${PV//_rc/-rc}"
+MY_LIBUV_V="35b1504507a7a4168caae3d78db54d1121b121e1"
+MY_UTF8PROC_V="5c632c57426f2e4246e3b64dd2fd088d3920f9e5"
+MY_LIBWHICH_V="81e9723c0273d78493dc8c8ed570f68d9ce7e89e"
+MY_DSFMT_V="2.2.3"
+
+DESCRIPTION="High-performance programming language for technical computing"
+HOMEPAGE="https://julialang.org/"
+SRC_URI="
+	https://github.com/JuliaLang/${PN}/releases/download/v${MY_PV}/${PN}-${MY_PV}.tar.gz
+	https://api.github.com/repos/JuliaLang/libuv/tarball/${MY_LIBUV_V} -> ${PN}-libuv-${MY_LIBUV_V}.tar.gz
+	https://api.github.com/repos/JuliaLang/utf8proc/tarball/${MY_UTF8PROC_V} -> ${PN}-utf8proc-${MY_UTF8PROC_V}.tar.gz
+	https://api.github.com/repos/vtjnash/libwhich/tarball/${MY_LIBWHICH_V} -> ${PN}-libwhich-${MY_LIBWHICH_V}.tar.gz
+	http://www.math.sci.hiroshima-u.ac.jp/~m-mat/MT/SFMT/dSFMT-src-${MY_DSFMT_V}.tar.gz -> ${PN}-dsfmt-${MY_DSFMT_V}.tar.gz
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	sys-devel/llvm:9=[llvm_targets_NVPTX(-)]
+"
+LLVM_MAX_SLOT=9
+
+RDEPEND+="
+	dev-libs/double-conversion:0=
+	dev-libs/gmp:0=
+	dev-libs/libgit2:0=
+	>=dev-libs/libpcre2-10.23:0=[jit,unicode]
+	dev-libs/mpfr:0=
+	dev-libs/openspecfun
+	>=net-libs/mbedtls-2.2
+	sci-libs/amd:0=
+	sci-libs/arpack:0=
+	sci-libs/camd:0=
+	sci-libs/ccolamd:0=
+	sci-libs/cholmod:0=
+	sci-libs/colamd:0=
+	sci-libs/fftw:3.0=[threads]
+	sci-libs/openlibm:0=
+	sci-libs/spqr:0=
+	sci-libs/umfpack:0=
+	sci-mathematics/glpk:0=
+	>=sys-libs/libunwind-1.1:7=
```

Comment:
This dependency is bad. I have changed it back to `:0=`

Reply:
Yes, it looks like that was changed in 192c9e77e35e91b3d03525ca5bab6d2e499e2f2c.